add-domain

```diff
@@ -999,3 +999,32 @@ wiflix.xyz
 wiflix.yachts
 coronavirus.zone
 fullcopec.zone
+dexsceerner.net
+app.dexscreener-home.net
```

Comment:
Double of `dexscreener-home.net`

- https://kb.mypdns.org/issue/MTX-127023/dexscreener-home.net MTX-127023
- https://mypdns.youtrack.cloud/issue/MTXB-209541/dexscreener-home.net MTXB-209541
- https://github.com/mypdns/matrix/issues/127023